backend/clap/src/main/java/zpi/ppea/clap/config/ValueConfig.java

```diff
@@ -17,4 +17,31 @@ public class ValueConfig {
     @Value("${firebase.audience}")
     private String firebaseAudience;
 
+    @Value("${firebaseAgent.auth_done}")
+    private String authDone;
+
+    @Value("${firebaseAgent.first_name}")
+    private String firstName;
+
+    @Value("${firebaseAgent.last_name}")
+    private String lastName;
```

Comment:
and those values are statically defiend in application.yaml...? perhaps `public static const String` would be enough or are we expecting that config to be changing often?

Reply:
It's better to store them in one place and not write "first_name" every time. The disadvantage of const String is that "first_name" will be in code and what if token claim changes? It's easier to manage them from application.yml because I don't care what's under FIRST_NAME and I don't see a reason to store this string in class.

Reply:
I agree to putting it to a variable, however i don't see this as configuration -- not any more than `"refresh" ` being a header name or prefix `"Bearer "` - shouldn't those be read from application.yaml as well?

Reply:
Yeah, good idea